src/tests/test_sysargv.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+import sys
+
+
+class SysArgvTests(unittest.TestCase):
+    """Test sys.argv state."""
+    def testSysArgvState(self):
+        """Test sys.argv state."""
+        argv = list(sys.argv) # need to make a copy of list
+        import clr
+        self.assertTrue(argv == sys.argv)
```

Comment:
`clr` is loaded before all tests being. its the 5th line on `run_tests.py` i think.